src/components/pages/ImageClassification/ImageUploadModal/ImageUploadModal.jsx

```diff
@@ -138,7 +143,11 @@ const ImageUploadModal = ({
 
       const result = await validateDimensions(file)
       if (!result.valid || result.corrupt) {
-        toast.error(`File is invalid or corrupt: ${file.name}`)
+        if (result.tooSmall) {
+          toast.error(`Image Too Small: ${file.name}`)
```

Comment:
Please add these to the language file, with sentence casing
```suggestion
          toast.error(`${language.imageClassification.imageTooSmall}: ${file.name}`)
```

```suggestion for language.ts
imageTooSmall: 'This photo is too small. Photos should be a minimum height and width of 1124 pixels."
```

Reply:
@hurrendor  beside adding the new text for image dimension, adding a file name as in screenshot is easier for user to find the invalid image. 
I also move other image upload errors to language.
![Screenshot 2025-05-23 at 14 34 33](https://github.com/user-attachments/assets/24eda932-3fee-44ff-be2d-3a798f447691)

Reply:
Nice! That flows much better.